src/index.ts

```diff
@@ -161,21 +163,27 @@ export default class IncludeFragmentElement extends HTMLElement {
         if (!isWildcard(this.accept) && (!ct || !ct.includes(this.accept ? this.accept : 'text/html'))) {
           throw new Error(`Failed to load resource: expected ${this.accept || 'text/html'} but was ${ct}`)
         }
-        return response
+        return response.text()
+      })
+      .then(data => {
+        // Dispatch `load` and `loadend` async to allow
+        // the `load()` promise to resolve _before_ these
```

Comment:
```suggestion
        // the `loadstart()` promise to resolve _before_ these
```

If I'm understanding this correclty, shouldn't it be `loadstart` here?

Reply:
This references the `load()` function. It is saying that the promise returned by the `load()` function should resolve before the `load`/`loadend` events are dispatched.